test/rules.json

```diff
@@ -1,30 +1,58 @@
 {
-  "version":"1.0",
-  "events":[
+  "version": "2.1",
+  "rules": [
     {
-      "id":"id",
+      "id": "id",
       "name": "some name",
-      "tags":{
-        "type":"security_scanner",
-        "crs_id":"0"
+      "tags": {
+        "type": "security_scanner",
+        "crs_id": "0",
+        "category": "attack_attempt"
       },
-      "conditions":[
+      "conditions": [
         {
-          "operation":"phrase_match",
-          "parameters":{
-            "inputs":[
-              "server.request.headers.no_cookies"
+          "parameters": {
+            "inputs": [
+              {
+                "address": "server.request.headers.no_cookies"
+              }
             ],
-            "list":[
+            "list": [
               "hello world"
             ]
-          }
+          },
+          "operator": "phrase_match"
         }
       ],
-      "transformers":[
+      "transformers": [
         "lowercase"
+      ]
+    },
+    {
+      "id": "nfd-000-001",
+      "name": "weird case",
+      "tags": {
+        "type": "security_scanner"
+      },
+      "conditions": [
+        {
+          "operator": "match_regex",
+          "parameters": {
+            "inputs": [
+              {
+                "address": "server.response.status"
+              }
+            ],
+            "regex": "^404$",
+            "options": {
+              "case_sensitive": true
```

Comment:
Without this line, the test I added passes. @vdeturckheim 🤔

Reply:
@Anilm3 @Taiki-San can tou confirm this is a valid option? Thanks a lot

Reply:
There is no support for booleans in bindings - fixing